troubadix/plugins/security_messages.py

```diff
@@ -14,34 +14,69 @@
 #
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-import re
 from pathlib import Path
 from typing import Iterator
 
 from troubadix.helper.patterns import ScriptTag, get_script_tag_pattern
 from troubadix.plugin import FileContentPlugin, LinterError, LinterResult
 
+security_message_implementations = [
```

Comment:
```suggestion
SECURITY_MESSAGE_IMPLEMENTATIONS = [
```
Because it's used as a constant